controller/user-controller.js

```diff
@@ -0,0 +1,146 @@
+const db = require("../models");
+const config = require("../config/auth.config");
+
+var jwt = require("jsonwebtoken");
+var bcrypt = require("bcryptjs");
+
+exports.signup = (req, res) => {
+  // Generate account No
+  const accountNo = () => {
+    let num = "";
+    while (num.length < 10) {
+      num += Math.floor(Math.random() * 10);
+    }
+    return num;
+  };
+  const generateAccountNo = accountNo();
```

Comment:
Why declare another const here when you can easily run accountNumber: accountNo(); ?

Reply:
completely modified the function